src/Compute/Compute/help/Set-AzVMDiskEncryptionExtension.md

```diff
@@ -48,8 +48,10 @@ Set-AzVMDiskEncryptionExtension [-ResourceGroupName] <String> [-VMName] <String>
 
 ## DESCRIPTION
 The **Set-AzVMDiskEncryptionExtension** cmdlet enables encryption on a running infrastructure as a service (IaaS) virtual machine in Azure.
-This cmdlet enables encryption by installing the disk encryption extension on the virtual machine.
-If no *Name* parameter is specified, an extension with the default name AzureDiskEncryption for virtual machines that run the Windows operating system or AzureDiskEncryptionForLinux for Linux virtual machines are installed.
+This cmdlet enables encryption by installing the disk encryption extension on the virtual machine. 
+
+The *VolumeType* parameter is required when encrypting Linux virtual machines, but can be omitted when encrypting Windows virtual machines.
```

Comment:
This change seems to omit the previous information about the Name parameter.  I'm not sure that needs to be included in the description, maybe it can be included later, but it is true that the default extension name used by this cmdlet will correspond to the type of operating system being targeted if not specified as something else by the user.

Reply:
I readded the info in the Name parameter section below.